Examples/MAX32572/AES/main.c

```diff
@@ -416,11 +416,11 @@ int main(void)
 
     if (fail == 0) {
         printf("Example Succeeded\n");
+        return 0;
     } else {
         printf("Example Failed\n");
+        return -1;
     }
 
-    while (1) {}
-
     return 0;
```

Comment:
This line might cause warning,  like unreachable point...

Reply:
@ozersa and @Jake-Carter   Can you build MAX32572 samples? When I import the projects on Eclipse I am seeing "Preprocessor Include Path/Entries/GNU C/CDT Cross GCC Built-in Compiler Settings" as empty. Therefore, the project can not find gcc command. This is why I have skipped MAX32572 samples.